app/renderer/js/pages/preference/connected-org-section.ts

```diff
@@ -27,21 +36,21 @@ class ConnectedOrgSection extends BaseSection {
 		`;
 	}
 
-	init() {
+	init(): void {
 		this.initServers();
 	}
 
-	initServers() {
+	initServers(): void {
 		this.props.$root.innerHTML = '';
 
 		const servers = DomainUtil.getDomains();
 		this.props.$root.innerHTML = this.template();
 
-		this.$serverInfoContainer = document.getElementById('server-info-container');
-		this.$existingServers = document.getElementById('existing-servers');
-		this.$newOrgButton = document.getElementById('new-org-button');
-		this.$addCertificateContainer = document.getElementById('add-certificate-container');
-		this.$findAccountsContainer = document.getElementById('find-accounts-container');
+		this.$serverInfoContainer = document.querySelector('#server-info-container');
```

Comment:
Can someone explain why we have moved to `querySelector` instead of `getElementById`?

Reply:
This is also an xo rule to prefer querySelector and querySelectorAll instead of getElementById and getElementByClass.